sklearn/svm/src/liblinear/linear.cpp

```diff
@@ -70,6 +78,33 @@ static void info(const char *fmt,...)
 static void info(const char *fmt,...) {}
 #endif
 
+// New function to ensure the same behaviour for random number generation on windows and linux
+int myrand() {
```

Comment:
I know it probably does nothing, but: `inline`?

Reply:
ok